assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/index.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames';
+import { format, isValid } from 'date-fns';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './style.css';
+import EspressoButton from '@appInputs/EspressoButton';
+
+export interface EditButtonProps {
+	tooltip: string;
+	tooltipPosition: string;
+}
+
+export enum Position {
+	LEFT = 'left',
+	RIGHT = 'right',
+}
+
+export interface BiggieCalendarDateProps {
+	date: Date;
+	editButton: EditButtonProps;
+	htmlClass: string;
+	headerText: string[] | string;
+	footerText: string[] | string;
+	onEdit: (event: React.MouseEvent<HTMLButtonElement>) => void;
+	position: Position;
+	showTime: boolean;
+}
+
+/**
+ * Displays a full calendar date, but REALLY BIG!!!
+ */
+export const BiggieCalendarDate: React.FC<BiggieCalendarDateProps> = ({
+	date,
+	htmlClass,
+	headerText,
+	footerText,
+	position = Position.LEFT,
+	onEdit = null,
+	editButton = {},
+	showTime = false,
+}) => {
+	if (!isValid(date)) {
+		return null;
+	}
+	const classes = classNames('ee-biggie-calendar-date-bg', htmlClass, {
+		'ee-bcd-pos-left': position === Position.LEFT,
+		'ee-bcd-pos-right': position !== Position.LEFT,
+	});
+	const editDateButton =
+		typeof onEdit === 'function' ? (
```

Comment:
maybe a personal preference, but I like more short circuit evaluation in this situations then ternary op

Reply:
I can do that